### ISSUE TITLE
Windows installer does not run

### ISSUE THREAD

**Issue by JorisGoosen** [jasp-stats/jasp-issues#2058]
Hello,
I have the same problem. I have windows 7 I know the support ended from 0.13 version but I had no problem till 0.16 but any version after that I try to install I'm faced with the same error. I tried this
```
cd <to/the/folder-where-you-unpacked>
jasp --safeGraphics --junctions 
```
for the zip version extract but it gave me "access denied" although I'm the administrator. Is there a workaround for this issue or it's related to my OS.
Thanks a lot in advance.

_Originally posted by @bassam-abulnoor in https://github.com/jasp-stats/jasp-issues/issues/1841#issuecomment-1471902850_

**Comment by JorisGoosen** [comment]
Well, @bassam-abulnoor you are very right that we do not support Windows 7 anymore... Neither does Microsoft, so Id strongly recommend either upgrading your windows version or installing linux instead.

However we can have a quick look whether or not we can fix your problem.

You did not mention what version of JASP you are installing, nor what errors you get, nor whether you see more info besides the access denied. I would need as much information as possible to even have a chance of helping you

**Comment by bassam-abulnoor** [comment]
![1678975537748](https://user-images.githubusercontent.com/51817217/225642641-aff62553-9d3a-4032-a6e9-fdf1330a59be.jpg)
 and this how the folder properties looks like 
![1678975655188](https://user-images.githubusercontent.com/51817217/225643308-6b8e3353-b07e-41da-ba5c-ffa4e00560e2.jpg)

**Comment by JorisGoosen** [comment]
What kind of drive is "I:" ?

I think junctions only work on NTFS, and probably not on FAT.

**Comment by bassam-abulnoor** [comment]
Each in a separate command and line? when I copy the code it's pasted as one continuous line

**Comment by bassam-abulnoor** [comment]
when I run the second line
cd JASP-0.17.1-64bit
I get "The system cannot find the path specified"

**Comment by JorisGoosen** [comment]
well change it to wherever you *did* unpack it

**Comment by JorisGoosen** [comment]
And are you on `I:` when you run it?
If you dont understand how the command prompt works, now worry, but then show me a screenshot otherwise I have to guess what you are doing

**Comment by bassam-abulnoor** [comment]
It worked! but it gave me this error
![1678976301085](https://user-images.githubusercontent.com/51817217/225646000-69812c19-3b17-4e17-89f6-f834a938f841.jpg)

**Comment by JorisGoosen** [comment]
Yes you might have to run the "vcredist" file that is also in that folder.

**Comment by JorisGoosen** [comment]
This error indicates you are missing a DLL, we can figure out later which one if running the `vcredist` installer doesnt help

**Comment by bassam-abulnoor** [comment]
When I run vcredist it tells me it's already installed. When I run jasp engine it tells me I'm messing R.dll

**Comment by JorisGoosen** [comment]
Try downloading this: https://www.dependencywalker.com/depends22_x86.zip

Open it and open "JASP.exe" with it. Then export the results to "Dependency Walker Image (DWI)" and upload here

**Comment by bassam-abulnoor** [comment]
[JASP.zip](https://github.com/jasp-stats/jasp-issues/files/10991939/JASP.zip)
The site won't accept "DWI" extension I I zipped it

**Comment by JorisGoosen** [comment]
Aaah... Your Windows 7 is 32bit I guess?

**Comment by bassam-abulnoor** [comment]
no 64

**Comment by JorisGoosen** [comment]
Can you show my your windows "about" ?

> To check your PC hardware specs, from the desktop find the icon that is labeled “My Computer”. Right-click on this and select Properties. A window should appear summarizing your PC hardware specs including processor, Memory (RAM), and other system info, including Windows version.

Another screenshot would be lovely

**Comment by bassam-abulnoor** [comment]
![1678977754923](https://user-images.githubusercontent.com/51817217/225653511-0d5ada32-54b6-4cde-a2b4-ee5a6f4b3a85.jpg)

**Comment by JorisGoosen** [comment]
Hmm, because the file you sent me is showing that its trying to load 32bit binaries for some reason... Ill have another look

**Comment by JorisGoosen** [comment]
Ah well, that isnt the problem. As it turns out, the main library/framework JASP is based on (Qt) really no longer supports Windows 7, and it gives exactly that error you get.

So Im afraid there is no way around it, you will either have to use an old version of JASP. Or use a different OS.
Do keep in mind that support for windows 7 was discontinued more than 3 years ago.

It is quite a bad idea to keep using it and Im sure other updated software will also not run in the future. Not even thinking about the risk of getting viruses/hacks/whatever.

Sorry I cant help you out further and thanks for the quick responses

**Comment by bassam-abulnoor** [comment]
Thanks a lot for your cooperation and patience I really appreciate it :)
